packages/react/src/components/Storage/FileUploader/Previewer/PreviewerIcons.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Icon } from '../../../../primitives';
+import { IconProps } from '../types';
+
+export const ErrorIcon = ({ className }: IconProps): JSX.Element => (
+  <Icon
+    pathData="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm1 15h-2v-2h2v2zm0-4h-2V7h2v6z"
+    className={className}
+  />
+);
+
+export const fileIcon = (
+  <Icon pathData="M6 2c-1.1 0-1.99.9-1.99 2L4 20c0 1.1.89 2 1.99 2H18c1.1 0 2-.9 2-2V8l-6-6H6zm7 7V3.5L18.5 9H13z" />
+);
+
+export const CloseIcon = ({ className }: IconProps): JSX.Element => (
+  <Icon
+    className={className}
+    pathData="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
+  />
+);
+
+export const EditIcon = ({ fontSize }: { fontSize: string }): JSX.Element => (
+  <Icon
+    fontSize={fontSize}
+    pathData="M3 21h3.75L17.81 9.94l-3.75-3.75L3 17.25V21zm2-2.92l9.06-9.06.92.92L5.92 19H5v-.92zM18.37 3.29a.996.996 0 00-1.41 0l-1.83 1.83 3.75 3.75 1.83-1.83a.996.996 0 000-1.41l-2.34-2.34z"
+  />
+);
+
+export const CheckIcon = ({ className }: IconProps): JSX.Element => (
+  <Icon
+    className={className}
+    color="font.success"
+    pathData="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8zm4.59-12.42L10 14.17l-2.59-2.58L6 13l4 4 8-8z"
+  />
+);
```

Comment:
More of a general note to me, but I'll be cleaning these icons up in another PR (adding them as svg to the ui package, and running the `build:icons` script to generate the tsx files in the Icon primitive folder.

Reply:
Thanks @dbanksdesign !